client/me/purchases/cancel-purchase/loading-placeholder.jsx

```diff
@@ -6,39 +6,35 @@ import React from 'react';
 /**
  * Internal dependencies
  */
+import Button from 'components/button';
 import Card from 'components/card';
 import CompactCard from 'components/card/compact';
-import Button from 'components/button';
 import LoadingPlaceholder from 'me/purchases/components/loading-placeholder';
 import { managePurchase } from 'me/purchases/paths';
 import titles from 'me/purchases/titles';
 
-const CancelPurchaseLoadingPlaceholder = React.createClass( {
-	propTypes: {
-		purchaseId: React.PropTypes.string.isRequired,
-		selectedSite: React.PropTypes.oneOfType( [
-			React.PropTypes.object,
-			React.PropTypes.bool
-		] ).isRequired
-	},
+const CancelPurchaseLoadingPlaceholder = ( { purchaseId, selectedSite } ) => {
+	return (
+		<LoadingPlaceholder
+			title={ titles.cancelPurchase }
+			path={ managePurchase( selectedSite.slug, purchaseId ) }>
+			<Card className="cancel-purchase-loading-placeholder__card">
+				<h2 className="loading-placeholder__content cancel-purchase-loading-placeholder__header" />
+				<div className="loading-placeholder__content cancel-purchase-loading-placeholder__subheader" />
+				<div className="loading-placeholder__content cancel-purchase-loading-placeholder__reason" />
+				<div className="loading-placeholder__content cancel-purchase-loading-placeholder__reason" />
+			</Card>
+
+			<CompactCard>
+				<Button className="cancel-purchase-loading-placeholder__cancel-button" />
+			</CompactCard>
+		</LoadingPlaceholder>
+	);
+};
 
-	render() {
-		return (
-			<LoadingPlaceholder
-				title={ titles.cancelPurchase }
-				path={ managePurchase( this.props.selectedSite.slug, this.props.purchaseId ) }>
-				<Card className="cancel-purchase-loading-placeholder__card">
-					<h2 className="loading-placeholder__content cancel-purchase-loading-placeholder__header" />
-					<div className="loading-placeholder__content cancel-purchase-loading-placeholder__subheader" />
-					<div className="loading-placeholder__content cancel-purchase-loading-placeholder__reason" />
-					<div className="loading-placeholder__content cancel-purchase-loading-placeholder__reason" />
-				</Card>
-				<CompactCard>
-					<Button className="cancel-purchase-loading-placeholder__cancel-button" />
-				</CompactCard>
-			</LoadingPlaceholder>
-		);
-	}
-} );
+CancelPurchaseLoadingPlaceholder.propTypes = {
+	purchaseId: React.PropTypes.string.isRequired,
+	selectedSite: React.PropTypes.object.isRequired
```

Comment:
If local storage is empty when we mount `/purchases/:site/:purchaseId/cancel`, `selectedSite` is false, and the page fails to load with [errors in the console](https://cloudup.com/c0Hx4Vk6Htr).